packages/starknet-snap/src/utils/logger.ts

```diff
@@ -32,35 +32,31 @@ export const emptyLog: LoggingFn = (
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   {};
 
-class Logger implements ILogger {
-  readonly log: LoggingFn;
+export class Logger implements ILogger {
+  log: LoggingFn;
 
-  readonly warn: LoggingFn;
+  warn: LoggingFn;
```

Comment:
What is the reason of removing the readonly ? 
- Is this to be able to change log on the fly if needed ? 
- Or since now log is set by the snap env file and not from ui this is not needed anymore ?

Reply:
prev code , readonly is incorrect, as we are updating in init()